scripts/apps/authoring/authoring/controllers/AssociationController.ts

```diff
@@ -28,49 +28,14 @@ export function AssociationController(config, content, superdesk,
         return isMediaEditable(config);
     };
 
-    /**
-     * @ngdoc method
-     * @name AssociationController#isImage
-     * @public
-     * @description Check if the rendition is image or not.
-     * @param {Object} rendition Rendition of the item.
-     */
-    this.isImage = function(rendition) {
```

Comment:
Can you keep it in this file? Utils are meant for generic code, not related to any features. You can export it from this file and import it where you need it.